src/Illuminate/Support/Number.php

```diff
@@ -115,13 +122,13 @@ public static function percentage(int|float $number, int $precision = 0, ?int $m
      * @param  string|null  $locale
      * @return string|false
      */
-    public static function currency(int|float $number, string $in = 'USD', ?string $locale = null)
+    public static function currency(int|float $number, string $in = '', ?string $locale = null)
     {
         static::ensureIntlExtensionIsInstalled();
 
         $formatter = new NumberFormatter($locale ?? static::$locale, NumberFormatter::CURRENCY);
 
-        return $formatter->formatCurrency($number, $in);
+        return $formatter->formatCurrency($number, ! empty($in) ? $in : static::$currency);
```

Comment:
I would handle it the same as the `$locale`-parameter:
```php
    public static function currency(int|float $number, ?string $in = null, ?string $locale = null)
    {
        static::ensureIntlExtensionIsInstalled();

        $formatter = new NumberFormatter($locale ?? static::$locale, NumberFormatter::CURRENCY);

        return $formatter->formatCurrency($number, $in ?? static::$currency);
```

Reply:
I went with `! empty()` in case `$in` was passed as empty :)